collector/src/api.rs

```diff
@@ -1,16 +1,19 @@
-pub mod next_commit {
+pub mod next_artifact {
     use database::Commit;
 
     #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
-    pub struct NextCommit {
-        pub commit: Commit,
-        pub include: Option<String>,
-        pub exclude: Option<String>,
-        pub runs: Option<i32>,
+    pub enum NextArtifact {
+        Release(String),
+        Commit {
+            commit: Commit,
+            include: Option<String>,
+            exclude: Option<String>,
+            runs: Option<i32>,
+        },
     }
 
     #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
     pub struct Response {
-        pub commit: Option<NextCommit>,
+        pub artifact: Option<NextArtifact>,
```

Comment:
Just a note: we've been careful about changing the API since we don't have the benefit of a compiler to tell us if all the users of this API have been properly updated. Are we sure that all consumers are aware of this change?

Reply:
@Mark-Simulacrum Are there any others notable users of this API than the perf. running `bench_next` script? FWIW, the API was already recently changed in https://github.com/rust-lang/rustc-perf/pull/1399.

Reply:
No, and I'm not worried about this API changing. It's not really useful unless you're trying to 1:1 match what the perf collector is doing and even then it's kind of a weird tool to use (compared to running your own database and frontend).